### PR TITLE
add render instructions, adjust documentation for cleaner config

### PR DESCRIPTION
**Pull request by mikeerickson**
- [x] Updated `README.md` configuration instructions 
- [x] Updated `README.md` to add `render` changes as defined in https://github.com/GrahamCampbell/Laravel-Exceptions/issues/107 issue

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for the PR. Going to sleep on this, then look at it again. :)

**Comment by ahme-d** [comment]
+1 for this

**Comment by mikeerickson** [comment]
@GrahamCampbell must be hibernating 👍

File: README.md [hunk 1]
```diff
@@ -43,12 +43,70 @@ $ composer require filp/whoops --dev
...
+  ...
+```  
+
+##### Laravel 5.1 / 5.2
+Modify `App\Exceptions\Handler` class to extend `GrahamCampbell\Exceptions\ExceptionHandler`
+
+##### Lumen
+Modify `App\Exceptions\Handler` class to extend `GrahamCampbell\Exceptions\LumenExceptionHandler`.
+
+####  Adjust Render Method
```
Comment (GrahamCampbell) [inline README.md:91]: I actually would not recommend editing the render method.

Comment (GrahamCampbell) [reply README.md:91]: This should be handled by writing a displayer.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GrahamCampbell) [reply README.md:91]: I really don't like laravel's native exception handler, and the way Laravel deals with errors across the board, by just saying "is json wanted", when actually, the correct thing to say is, "what does the client prefer out of the content types that i can respond with".

**Pull request closed without merging**